Tensile/KernelWriterAssembly.py

```diff
@@ -6482,24 +6492,32 @@ def closeSumAtLeastUnroll(self, kernel, prefetch, isOptNLL, isNGLL):
             if kernel["ExpandPointerSwap"]:
               kStr += self.getLabelDef("SkipLroSwap", "Skip LRO Swap\n")
 
-          kStr += self.comment1("Stores for OptNLL")
-          kStr += self.endSummation(kernel)
+            # Jump to Summation
+            kStr += inst("s_branch", "%s"%endSumLabel, "jump to Summation End")
+            kStr += "\n"
+            # Append label for pure OptNLL (no PAP interleaved)
+            kStr += "%s:\n" % self.getNamedLabel("SkipTo_PureOptNLL_LastTile")
```

Comment:
After this "OptNLL + PAP-interleaved", branch to Summation_End (Skip the following "OptNLL + noPAP" ). Add a label for Pure-OptNLL